fontc/src/args.rs

```diff
@@ -101,7 +110,7 @@ impl Args {
         Args {
             glyph_name_filter: None,
             source,
-            emit_ir: true,
+            incremental: true,
```

Comment:
Note that even if it's off by default, I decided to keep incremental=true when initialising Args::for_test, since many tests expect to be able to look inside the build/ directory for IR files.